lib/swagger.js

```diff
@@ -0,0 +1,49 @@
+var utils = require('../utils');
+var request = require('request');
+var fs = require('fs');
+var path = require('path');
+
+exports.swagger = true;
+exports.login = true;
+exports.desc = 'Upload your swagger file to ReadMe';
+exports.category = "services";
+
+exports.run = function(config, info, cb) {
+  if (!cb) cb = (err) => {
```

Comment:
Probably fine, but we should watch for people complaining that they're on an old vrsion of Node and this doesn't work.

Reply:
👍 just looked this up, arrow functions have been in node since 4, which is coming to it's end of life for support at the end of april